### DISCUSSION TITLE
Renaming Quick actions

### DISCUSSION THREAD

**Discussion by ERYpTION** [osmandapp/OsmAnd#12630] [Q&A]
I have renamed a few of my Quick actions but even though they now show correctly with their new names in the list (Configure screen - Quick actions) they still have the old names when I bring up the Quick action widget on the map. What do I need to do for the name changes to take effect?

**Comment by Chumva** [comment]
@ERYpTION Can you specify what actions you tried to rename? 
It is possible that you are using switchable actions, for example, "change map style", show the name of the next item which will be applied after selection.

**Reply by ERYpTION** [reply]
@Chumva It is map overlays. Some of them I am renaming because their standard names are to long to display in the Quick actions on the map. E.g. 'Waymarkedtrails Hiking' and 'Waymarkedtrails Cycling' both only show as "Waymarkedtrails" making them hard to distinguish.

**Comment by ERYpTION** [comment]
With the latest update the overlay quick actions for some reason now all show the name of the currently active one making it guesswork selecting another ... sigh ..
